app/app.tsx

```diff
@@ -4,27 +4,59 @@
  *
  * @format
  */
-import { StatusBar, View, type ViewStyle } from "react-native"
+import { DevSettings, NativeModules, StatusBar, View, type ViewStyle } from "react-native"
 import { connectToServer } from "./state/connectToServer"
 import { useTheme, useThemeName, withTheme } from "./theme/theme"
-import { useEffect } from "react"
+import { useEffect, useMemo } from "react"
 import { TimelineScreen } from "./screens/TimelineScreen"
-import { AppHeader } from "./components/AppHeader"
 import { useMenuItem } from "./utils/useMenuItem"
+import { Titlebar } from "./components/Titlebar"
+import { Sidebar } from "./components/Sidebar/Sidebar"
+import { useSidebar } from "./state/useSidebar"
+import { AppHeader } from "./components/AppHeader"
 
 if (__DEV__) {
   // This is for debugging Reactotron with ... Reactotron!
   // Load Reactotron client in development only.
   require("./devtools/ReactotronConfig.ts")
 }
 
-const menuConfig = {
-  remove: ["File", "Edit", "Format"],
-}
-
 function App(): React.JSX.Element {
   const [theme] = useThemeName()
   const { colors } = useTheme(theme)
+  const { toggleSidebar } = useSidebar()
+
+  const menuConfig = useMemo(
+    () => ({
+      remove: ["File", "Edit", "Format"],
+      items: {
+        View: [
+          {
+            label: "Toggle Sidebar",
+            shortcut: "cmd+b",
+            action: toggleSidebar,
+          },
+          ...(__DEV__
+            ? [
+                {
+                  label: "Toggle Dev Menu",
+                  shortcut: "cmd+shift+d",
+                  action: () => NativeModules.DevMenu.show(),
+                },
+              ]
+            : []),
```

Comment:
@leonskim Should we allow for falsey values and just ignore them? That way we could write:

```suggestion
          __DEV__ && {
            label: "Toggle Dev Menu",
            shortcut: "cmd+shift+d",
            action: () => NativeModules.DevMenu.show(),
          },
```

Reply:
Awesome!